### ISSUE TITLE
Display Single Event Component per day in Month View

### ISSUE THREAD

**Issue by nataze** [intljusticemission/react-big-calendar#1213]
First off I wanna say this is hands down the best events calendar out there in React!

Currently I have a custom event component that is displayed for each event. So if I have 8 events in a day, I get 8 events components.

But I'm trying to implement a summary event component that displays just once and shows the total number of events/bookings in a day within the month view, . Similar to the image below. 

![calendar](https://user-images.githubusercontent.com/13364089/52404812-c4f2c080-2a7e-11e9-997b-c5f1d0e05229.png)

Would this be possible to implement?

**Comment by drydenwilliams** [comment]
Do you mean the little "show more" button like below?

<img width="163" alt="screenshot 2019-02-08 at 10 25 35" src="https://user-images.githubusercontent.com/4403089/52472727-e9888e80-2b8b-11e9-8159-31049f615762.png">

**Comment by nataze** [comment]
I wanna display just one event component per day, a count of appointments for that day. If I can hide the “show more” button and display just one component that would work :)

**Comment by perlavianey** [comment]
@nataze were you able to do this? I need something similar :)

**Comment by Paolo-Rzf** [comment]
> 
> 
> Do you mean the little "show more" button like below?
> 
> <img alt="screenshot 2019-02-08 at 10 25 35" width="163" src="https://user-images.githubusercontent.com/4403089/52472727-e9888e80-2b8b-11e9-8159-31049f615762.png">

Can you help me? I want to display single event component per day in month view with the little "show more" button.

**Comment by bennyfink** [comment]
@Paolo-Rzf did you figure out how to do this?

**Comment by bennyfink** [comment]
If anyone else runs into this issue, one way (but not the most preferable) to accomplish this is to set a `height` attribute in the react-big-calendar css file for `.rbc-event`. This will trick the rbc code into thinking there is more/less space in the day to add events and will render the amount of events it thinks will fit based on the `.rbc-event` height. For example, if you set the height to be almost as tall as the day boxes, 1 event will render and the little "show more" button will appear as seen in the picture drydenwilliams sent. If anyone knows of a more elegant way to do this, let us know!

**Comment by ArtemGoldsmith** [comment]
Anyone has solution for this?

**Comment by dawidblocherultimatesystems** [comment]
I added height to only class .rbc-event and it works

**Comment by popcouncilconsulting** [comment]
> This will trick the rbc code into thinking there is more/less space in the day to add events and will render the amount of events it thinks will fit based on the `.rbc-event`

show us the code sample .... not working for me

**Comment by shbhmshekhar** [comment]
Try with  copying the whole style of event.scss rom react-big-calendar into your custom stylesheet and add a height property inside .rbc-event class. It is working for me

**Comment by paulbiswajit880** [comment]
Can we add image instead of text?

**Comment by amitsaini1807** [comment]
how can we modify number of events shows in a slot in month view, currently it shows 1 task and then more button, but i want 2 task to display and then more button, can anyone help this

**Comment by dotdotot** [comment]
I'd like to add a comment to share the solution I found for this issue. I'm using Tailwind CSS and set the height of the event in the renderEvent part with h-[18px]. This change appeared to work fine on mobile (Galaxy, iPhone), iPad, and desktop, with the "+ More" link displaying correctly without cutting off any event content. However, on the Galaxy Tab, it was observed that the "+ More" link was being clipped by other events.


The issue was that even though I added a height value in renderEvent, the internal calculations of the BigCalendar did not take this new height value into account when determining the display for "+ More". By adding height:18px !important to .rbc-event, I was able to resolve the issue and "+ More" displayed correctly.


To summarize: when setting a height value in renderEvent, it's necessary to also add the same height value to .rbc-event to ensure that the BigCalendar layout calculates the heights correctly.

**Comment by AXLShorts** [comment]
> > Do you mean the little "show more" button like below?
> > <img alt="screenshot 2019-02-08 at 10 25 35" width="163" src="https://user-images.githubusercontent.com/4403089/52472727-e9888e80-2b8b-11e9-8159-31049f615762.png">
> 
> Can you help me? I want to display single event component per day in month view with the little "show more" button.

Did you figure this out?

**Comment by BarrozoL** [comment]
Did anyone find a solution to this? 
The .rbc-row-content .rbc-row:nth-child(n + 3) { display: none; }  seems like a good fix, but for me it also hides the show more button...

**Comment by Kamronbekdevoloper** [comment]
@nataze    
.rbc-row {
   padding-top: 8px !important;
} 
this code for showing "x show more".  i write margin but it's same problem  after i write with "padding" and is working
